Packs/CoreAlertFields/IncidentFields/incidentfield-Category_Name.json

```diff
@@ -0,0 +1,142 @@
+{
+    "id": "incident_categoryname",
+    "version": -1,
+    "modified": "2022-01-12T08:13:44.844285695Z",
+    "ownerOnly": false,
+    "cliName": "categoryname",
+	"description": "Alert category based on the alert source. An example of an XDR Agent alert category is Exploit Modules.",
+    "name": "Category Name",
+    "type": "shortText",
+    "closeForm": false,
+    "editForm": true,
+    "required": false,
+    "neverSetAsRequired": false,
+    "isReadOnly": false,
+    "useAsKpi": false,
+    "locked": false,
+    "system": false,
+    "content": true,
+    "group": 0,
+    "hidden": false,
+    "openEnded": false,
+    "associatedToAll": true,
+    "unmapped": false,
+    "unsearchable": true,
+    "caseInsensitive": true,
+    "sla": 0,
+    "threshold": 72,
+    "fromVersion": "6.5.0",
+	"x2_fields": "alert_category",
```

Comment:
I think it's just "category". Why did you add the prefix?

Reply:
It's "alert_category" 100%
A screenshot from the python file with all the fields:
![image](https://user-images.githubusercontent.com/81556849/181509999-34183921-d9db-4857-b607-723c4d0085d1.png)

I think the issue with this field was that we already had an incident field with the name "Alert Name" in XSOAR (under common types)